bin/install-prereqs.sh

```diff
@@ -3,8 +3,8 @@
 set -ex
 
 # Install linters
-go get -u github.com/alecthomas/gometalinter
-gometalinter --install --update --vendored-linters
+go get -u gopkg.in/alecthomas/gometalinter.v1
+gometalinter.v1 --install --update --vendored-linters
```

Comment:
You can always pin to a specific version without having to pin to such an old version by using a git sha:

https://github.com/docker/cli/blob/master/dockerfiles/Dockerfile.lint#L6-L9

Reply:
Will that install vendored listeners at the right SHA, too?
Seems like a docker image is probably cleanest way to accomplish this.